### ISSUE TITLE
Observation post idea: Devil in disguise

### ISSUE THREAD

**Issue by ViolentBeetle** [MoreEventsMod/More_Events_Mod_Beta#394]
And beginning idea I have, could use some suggestions on resolution.

If observation post is set to agressive studies, one day a specimen you kidnapped dies and you find out he's an alien. Apparently, someone somewhere was infiltrating the planet but were abandoned (FE maybe?). 

One possible outcome - you identify infiltrators and contact them, either bargaining to return to the mission and prepare for annexation in exchange for return to spacefaring lifestyle or rescuing them and gaining a colonization ship.

But I'd like to have more.

**Comment by Goldziher** [comment]
interesting.

This is an idea I'm implementing in a way in a seperate mod, but it could be done here. How about creating a special observation post mission that gives you extra benefits, e.g. extra research.

Content-wise, you could be observing the interaction of two different alien species, and the slow war that is beginning between them. You could also interfere in this war in various ways.

**Comment by ViolentBeetle** [comment]
Special mission won't do because I can't disable the basic two.

Regular observation events are duable of course, the question is, how.

**Comment by Goldziher** [comment]
are you sure? can't you do something like:
potential = {
	has_country_flag = xyz_flag
}

or you simply don't want to overwrite the vanilla file?

anyhow, the simplest way would be a multiple choice event with forking possibilities.

**Comment by ViolentBeetle** [comment]
>or you simply don't want to overwrite the vanilla file?

Yes. I wouldn't want to interfere with compatibility. Forking events are the best way indeed, but what should it be is what I'm brainstorming.

**Comment by Goldziher** [comment]
I see.

Well, how about you begin with the first event in the chain and we'll see what options we can come up with?
